fastlane-env-reminder/docs/contributors.md

```diff
@@ -1,29 +0,0 @@
-# Contributors
```

Comment:
Was this file be removed by accident?

Reply:
Nope, it was removed on purpose in favour of https://github.com/fastlane/github-actions/blob/master/CONTRIBUTING.md :wink: